### ISSUE TITLE
`prefer-bigint-literal`: Support number with `+`/`-` sign

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#2733]
### Description

I wasn't aware that `BigInt('-1')` and `BigInt('+1')` is also valid.

### Examples

```js
// ❌
BigInt('-1')

// ✅
-1n
```

```js
// ❌
BigInt(-1)

// ✅
-1n
```

```js
// ❌
BigInt('+1')

// ✅
1n
```

### Additional Info

The string cases should already supported, need tests.
The negative integer case should be fixed.
